package.json

```diff
@@ -2,8 +2,30 @@
   "name": "radiance-ui",
   "version": "17.2.3",
   "description": "Curology's React based component library",
-  "main": "dist/bundle.js",
-  "module": "dist/bundle-es/index.js",
+  "main": "./dist/cjs/index.cjs",
+  "module": "./dist/esm/index.js",
+  "exports": {
+    ".": {
+      "browser": "./dist/esm/index.js",
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.cjs"
+    },
+    "./lib/constants": {
```

Comment:
if we have these resolutions now, would it make sense to cleanup the below `files` options??? so they are not spit out as is in the package

Reply:
I think the files is still necessary if you want 'go to definition' and Typescript types to work?